Foosball/FoosballWebsite/Core/FeedEngine.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using LiveGameFeed.Data.Abstract;
+using LiveGameFeed.Models;
+using Microsoft.Extensions.Logging;
+using RecurrentTasks;
+
+namespace LiveGameFeed.Core
+{
+    public class FeedEngine : IRunnable
+    {
+        private ILogger logger;
```

Comment:
You should use _logger for privates, and remove "this" from "this.logger". Also in other places